civictechprojects/static/css/partials/_ProjectFilterContainer.scss

```diff
@@ -138,3 +138,12 @@
     margin: 0;
   }
 }
+
+.favorite-filter {
+  font-weight: bold;
+  padding-left: 5px;
```

Comment:
I did a few iterations on this to figure out how best to display it, heart on the left, heart on the right, and what felt best to me was Favorites on the left and then the heart icon, but not all the way over on the far right like where the dropdown chevron icons are.

Like so:
![image](https://user-images.githubusercontent.com/16870466/134580913-cd4aa732-df2d-4b50-93a0-15af863b790f.png)

To do that,
```css
.favorite-filter {
  font-weight: bold;
  padding: 12px 5px;
  display: flex;
  align-items: center;
  }
```

We will also need to add .favorite-filter to the rules which also add 15px margin on mobile, or else we get this problem:
![image](https://user-images.githubusercontent.com/16870466/134582218-1a94480f-1e5a-4f3d-b615-cda866737fdb.png)


To do that, see here:
https://github.com/DemocracyLab/CivicTechExchange/blob/d5f9b6c6b9e76ed60c7d7805b71f9932656acecc/civictechprojects/static/css/partials/_ProjectFilterContainer.scss#L109-L112

and here:
https://github.com/DemocracyLab/CivicTechExchange/blob/d5f9b6c6b9e76ed60c7d7805b71f9932656acecc/civictechprojects/static/css/partials/_ProjectFilterContainer.scss#L135-L140

Adding favorite-filter as another class to both existing rules should do it.